### PR TITLE
Issue 2 user can view their entry

### PR DESCRIPTION
**Pull request by dennymin**
issue 2 user can view their entry

### CODE REVIEW

**Comment by dennymin** [comment]
issue 2 user can view their entry 2

File: js/main.js [hunk 1]
```diff
@@ -12,16 +12,101 @@ $photoUrl.addEventListener('input', updateSRC);
...
+var $entriesLink = document.querySelector('.header-entries-link');
+function goToEntriesViaLink(event) {
+  $newEntryForm.setAttribute('class', 'hidden');
+  $entriesList.setAttribute('class', 'entries-list');
+  data.view = 'entries';
+}
+$entriesLink.addEventListener('click', goToEntriesViaLink);
+
+// prevent page from refreshing
+function preventRefresh(event) {
```
Comment (codingcodymiller) [inline js/main.js]: This isn't preventing refreshing, it is just showing the last view that the user visited. You should rename this to something like `showLastViewVisited`

File: js/main.js [hunk 2]
```diff
@@ -12,16 +12,101 @@ $photoUrl.addEventListener('input', updateSRC);
...
+$newEntryButton.addEventListener('click', goToNewForm);
+
+// nav links
+var $entriesLink = document.querySelector('.header-entries-link');
+function goToEntriesViaLink(event) {
+  $newEntryForm.setAttribute('class', 'hidden');
+  $entriesList.setAttribute('class', 'entries-list');
+  data.view = 'entries';
+}
+$entriesLink.addEventListener('click', goToEntriesViaLink);
```
Comment (codingcodymiller) [inline js/main.js]: You would benefit from making a single function named something like `switchView` that receives an argument of the name of the view to switch to, and the function should show that view and hide all other views. That way, you won't need to make a new function for each new view created.

File: js/main.js [hunk 3]
```diff
@@ -12,16 +12,101 @@ $photoUrl.addEventListener('input', updateSRC);
 
 // save button interactions code
 var $form = document.forms[0];
-function retrieveData(event) {
+var $newEntryForm = document.querySelector('.new-entry-form');
```
Comment (codingcodymiller) [inline js/main.js]: Doesn't the `$form` variable hold the same element as this?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js [hunk 4]
```diff
@@ -78,35 +77,35 @@ function generateDOM(event) {
...
-    data.view = 'entries';
-  } else if (currentView === 'entry-form') {
-    $newEntryForm.setAttribute('class', 'new-entry-form');
-    $entriesList.setAttribute('class', 'hidden');
-    data.view = 'entry-form';
+function showLastViewVisited(event) {
+  if (data.view === 'entries') {
+    switchViewTo('entries');
+  } else if (data.view === 'entry-form') {
+    switchViewTo('entry-form');
```
Comment (codingcodymiller) [inline js/main.js]: You should just be able to do this: `switchViewTo(data.view)`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**